From 0cfd58e0656be7ed32f4274b65dd184ee358a865 Mon Sep 17 00:00:00 2001
From: Marek Rusinowski <marekrusinowski@gmail.com>
Date: Mon, 17 Apr 2023 13:17:00 +0200
Subject: [PATCH] Remove launcher self-update

---
 src/launcher_wizard.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/launcher_wizard.js b/src/launcher_wizard.js
index 82c418f..750042f 100644
--- a/src/launcher_wizard.js
+++ b/src/launcher_wizard.js
@@ -149,7 +149,7 @@ class Wizard extends EventEmitter {
 
 			// Queue asynchronous check for launcher update.
 			const isDev = !require('electron').app.isPackaged;
-			if (!isDev) {
+			if (false && !isDev) {
 				const asyncLauncherUpdateCheck = {
 					promise: null,
 					action: () => {
@@ -219,7 +219,7 @@ class Wizard extends EventEmitter {
 					}
 				});
 			} else {
-				console.log('Development version: no self-update required');
+				// console.log('Development version: no self-update required');
 			}
 		}
 
-- 
2.40.0

