From d3052a6a60f5e278b887fd59422e4cbc9c775841 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tobias=20Sch=C3=B6nberg?= <tobias47n9e@gmail.com>
Date: Thu, 6 Jan 2022 23:16:38 +0100
Subject: [PATCH] Remove launcher self-update

---
 src/launcher_wizard.js | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/src/launcher_wizard.js b/src/launcher_wizard.js
index c8d67d9..b8597c6 100644
--- a/src/launcher_wizard.js
+++ b/src/launcher_wizard.js
@@ -54,23 +54,6 @@ class Wizard extends EventEmitter {
 				});
 			}
 
-			steps.push({
-				name: 'launcher_update',
-				action: () => {
-					const isDev = require('electron-is-dev');
-					log.info('Checking for launcher update');
-					if (!isDev) {
-						updater.checkForUpdates();
-					} else {
-						console.log('Development version: no self-update required');
-						setTimeout(() => {
-							this.nextStep();
-						}, 300);
-					}
-				}
-			});
-
-
 			config.downloads.engines.forEach((engine) => {
 				steps.push({
 					name: 'engine',
-- 
2.32.0

