From 3c8803e16cfb592a1256d3b20a2efc0017df73d2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tobias=20Sch=C3=B6nberg?= <tobias47n9e@gmail.com>
Date: Thu, 12 May 2022 21:11:35 +0200
Subject: [PATCH] Remove launcher self-update

---
 src/launcher_wizard.js | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/src/launcher_wizard.js b/src/launcher_wizard.js
index 0035d90..b8597c6 100644
--- a/src/launcher_wizard.js
+++ b/src/launcher_wizard.js
@@ -54,23 +54,6 @@ class Wizard extends EventEmitter {
 				});
 			}
 
-			steps.push({
-				name: 'launcher_update',
-				action: () => {
-					const isDev = !require('electron').app.isPackaged;
-					log.info('Checking for launcher update');
-					if (!isDev) {
-						updater.checkForUpdates();
-					} else {
-						console.log('Development version: no self-update required');
-						setTimeout(() => {
-							this.nextStep();
-						}, 300);
-					}
-				}
-			});
-
-
 			config.downloads.engines.forEach((engine) => {
 				steps.push({
 					name: 'engine',
-- 
2.36.1

