From 3735f9c12ea205c8cafdcefe5cc213b4916779f3 Mon Sep 17 00:00:00 2001
From: Marek Rusinowski <marekrusinowski@gmail.com>
Date: Sat, 8 Oct 2022 21:00:46 +0200
Subject: [PATCH] Remove launcher self-update

---
 src/launcher_wizard.js | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/src/launcher_wizard.js b/src/launcher_wizard.js
index 874880d..656a539 100644
--- a/src/launcher_wizard.js
+++ b/src/launcher_wizard.js
@@ -54,22 +54,6 @@ class Wizard extends EventEmitter {
 				});
 			}
 
-			steps.push({
-				name: 'launcher_update',
-				action: () => {
-					const isDev = !require('electron').app.isPackaged;
-					log.info('Checking for launcher update');
-					if (!isDev) {
-						updater.checkForUpdates();
-					} else {
-						console.log('Development version: no self-update required');
-						setTimeout(() => {
-							this.nextStep();
-						}, 300);
-					}
-				}
-			});
-
 			config.downloads.resources.forEach((resource) => {
 				steps.push({
 					name: 'resource',
-- 
2.37.2

